src/BoundedInts.dfy

```diff
@@ -34,5 +34,348 @@ module BoundedInts {
   newtype nat16 = x: int  | 0 <= x < 0x8000
   newtype nat32 = x: int  | 0 <= x < 0x8000_0000
   newtype nat64 = x: int  | 0 <= x < 0x8000_0000_0000_0000
+   
```

Comment:
I think we can omit all these changes once we remove the other files I'm suggesting we remove. We may just want to define one or two comparison functions in `examples` files instead.